client/my-sites/plan-features/style.scss

```diff
@@ -342,6 +342,11 @@ $plan-features-sidebar-width: 272px;
 	@include breakpoint( "<660px" ) {
 		padding: 20px 20px 0 20px;
 	}
+
+	strong {
```

Comment:
I would suggest adding a modifier class, esp since this changes the display type.